src/api/app/models/kiwi/repository.rb

```diff
@@ -3,7 +3,7 @@ class Kiwi::Repository < ApplicationRecord
   #### Includes and extends
 
   #### Constants
-  REPO_TYPES = ['apt-deb', 'rpm-dir', 'rpm-md', 'yast2'].freeze
+  REPO_TYPES = ['rpm-md', 'rpm-dir', 'yast2', 'apt-deb'].freeze
```

Comment:
Depending on which PR gets merged first, please make sure to not introduce the bug again:
https://github.com/openSUSE/open-build-service/pull/3862

Reply:
@ChrisBr, I will take care of this, thanks

Reply:
Don't worry, there tests that rely on "rpm-md" being the default one, and I guess @DavidKang did the same for the "rpm-dir" and "yast2" values...

Reply:
Thanks, just wanted to point it out :smile: